sdk/python/kfp/registry/registry_client.py

```diff
@@ -129,8 +129,7 @@ def _is_ar_host(self) -> bool:
         Returns:
             Whether the host is on Artifact Registry.
         """
-        # TODO: Handle multiple known hosts.
-        return self._known_host_key == 'kfp_pkg_dev'
+        return self._known_host_key in _KNOWN_HOSTS_REGEX.keys()
```

Comment:
_is_ar_host is specifically for checking if the host is Artifact Registry.. It should just check if `self._known_host_key == 'kfp_pkg_dev'`.

Reply:
Right. I reinstated the original code here.